src/worker/worker.ts

```diff
@@ -146,6 +146,12 @@ export class ExtensionWorker implements ExtensionWorkerTransport {
     return files.map(([project, spec]) => [project.config.name || '', spec])
   }
 
+  public getBrowserDebugOptions(): (Pick<ResolvedBrowserOptions, 'enabled' | 'provider'> & { project: string })[] | undefined {
```

Comment:
I think it might be better to just return this in https://github.com/vitest-dev/vscode/blob/f0fa52bbb4acd425374b43972ebb8372431d8706/src/worker/index.ts#L51

Then you can store it in https://github.com/vitest-dev/vscode/blob/8fbf0ea83276472ca78eec464962ae0ec3152449/src/api.ts#L398

And get the values at any point